next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx

```diff
@@ -0,0 +1,254 @@
+---
+description: Easily integrate with your exchange, custody solution, or product using the IOTA SDK.
+sidebar_label: Integrate IOTA SDK With Your Exchange
+keywords:
+  - integrate
+  - exchange
+  - account model
+  - addresses
+  - IOTA SDK
+  - setup
+  - NodeJS
+  - explanation
+---
+
+import WarningUnlockConditions from '../_admonitions/_warning-unlock-conditions.md';
+
+# Stardust Exchange Integration Guide
+
+:::warning
+
+The stardust update allows for complex tokenization schemes that can lead to monetary losses if misused.
+Transaction outputs may have multiple unlocking conditions which may require returning some or all of the amount, which could expire if not claimed in time, or which may not be unlockable for a very long time. You can lose money if you do not check the unlock conditions before accepting a deposit!
+
+:::
+
+## Integration Guide
+
+This guide explains how to integrate the IOTA SDK into your exchange, custody solution, or product.
+
+Features of the IOTA SDK:
+
+- Secure seed management.
+- Account management with multiple accounts and multiple addresses.
+- Confirmation monitoring.
+- Deposit address monitoring.
+- Backup and restore functionality.
+
+### How Does it Work?
+
+The IOTA SDK is a stateful package with a standardized interface for developers to build value transaction applications. It offers abstractions to handle payments and can optionally interact with [Stronghold](https://wiki.iota.org/stronghold.rs/getting_started) for seed handling, seed storage, and state backup.
```

Comment:
I don't know about the first sentence. Sounds a bit weird to me tbh. I probably would use the term `library` instead of `package`. Also I wouldn't say it's `standardized`. Do you mean `consistent`?

Reply:
I don't know, didn't write this, but I can change

Reply:
Changed